lighthouse-cli/bin.ts

```diff
@@ -229,11 +234,21 @@ function lighthouseRun(addresses: Array<string>, config: Object, flags: Object)
   return lighthouse(address, flags, config)
     .then((results: Results) => Printer.write(results, outputMode, outputPath))
     .then((results: Results) => {
+      const filename = `./${assetSaver.getFilenamePrefix({url: address})}.report.html`;
+
       if (outputMode === Printer.OutputMode[Printer.OutputMode.pretty]) {
-        const filename = `./${assetSaver.getFilenamePrefix({url: address})}.report.html`;
         Printer.write(results, 'html', filename);
       }
 
+      // Generate report.html, host it and open it in the default browser
+      if (flags.view) {
+        server.listen().then((port: number) => {
+          const fileAbsPath = `${__dirname}/server/reports/${filename}`;
+          Printer.write(results, 'html', fileAbsPath);
```

Comment:
ah gotcha. now i see what you were thinking here.

one way we could handle this is append a `<script>...</script>` to the end of the response HTML as its being served. (that's also how livereload servers work, just injecting scripts into the end of the file)

generally this has some advantages, in that we can keep the reporting stuff really consistent between our static and interactive reports.

wdyt?